dashboard/scripts/archive/backfill_script_level_ids_teacher_feedbacks_4.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# Backfill existing TeacherFeedbacks to set script_level_id based on script
+# stability. Scripts have is_stable set to true when we launch curriculum.
+# Scripts that do not have is_stable set to true are used internally, usually
+# as drafts.
+
+require_relative '../../config/environment'
+
+def feedbacks_without_script_level_id
+  TeacherFeedback.where(script_level_id: nil)
```

Comment:
This might go faster if you do `TeacherFeedback.includes(level: :script_levels).where(...)`, since it will mean you only have to do one DB read instead of 12,000.